src/main/java/net/fabricmc/loom/extension/MixinApExtensionImpl.java

```diff
@@ -79,7 +78,7 @@ private String getDefaultMixinRefmapName() {
 
 	@Override
 	protected PatternSet add0(SourceSet sourceSet, Provider<String> refmapName) {
-		PatternSet pattern = new PatternSet().setIncludes(Collections.singletonList("*.mixins.json"));
+		PatternSet pattern = new PatternSet().setIncludes(Collections.singletonList("*.json"));		// TODO: should I use `*.json` or `*`?
```

Comment:
Need input for this @modmuss50 @Juuxel

Reply:
`*.json` is fine imo, the files can't be anything else.